cmd/utils/flags.go

```diff
@@ -49,6 +49,12 @@ var (
 			Value:   path.Join(os.Getenv("HOME"), "/.kube/config"),
 			EnvVars: []string{"KUBERNETES_KUBECONFIG_PATH"},
 		},
+		&cli.StringFlag{
+			Name:    "kubernetes.manifest.dir",
+			Usage:   "The path of manifests",
+			Value:   "./deployments",
+			EnvVars: []string{"KUBERNETES_MANIFEST_DIR"},
```

Comment:
I'm sorry. It looks great the name is `KUBERNETES_MANIFEST_PATH` more. 😢
```suggestion
			EnvVars: []string{"KUBERNETES_MANIFEST_PATH"},
```

Reply:
i fixed it! thanks!